### ISSUE TITLE
Calendar Keros issue - Ensure buttons have discernible text

### ISSUE THREAD

**Issue by DhirajKhot** [OfficeDev/office-ui-fabric-react#11748]
### Environment Information

- **Package version(s)**: 6.210.5

### Describe the issue:

Calendar month navigations fail accessibility fastpass.
Please check the bug created in the past for the same - 
https://github.com/OfficeDev/office-ui-fabric-react/issues/8285

I checked the comment regarding no repro in v7. 
Can we have the fix ported to v6? 
We are using v6 and have no immediate plans to upgrade to v7.

#### Actual behavior:

Keros tool reports error

![image](https://user-images.githubusercontent.com/20378231/72794878-b63af480-3c62-11ea-9d8d-853a1b3cf985.png)

![image](https://user-images.githubusercontent.com/20378231/72794838-a4f1e800-3c62-11ea-9ea9-9c0a827f4f42.png)

#### Expected behavior:

Keros test pass should be successful

**Comment by khmakoto** [comment]
Hi @DhirajKhot, I just ran the "Accessibility Insights for Web" tool on our website using the Fabric 6 version and this error is not showing, are you sure you're using the latest version of Fabric 6?

**Comment by DhirajKhot** [comment]
Hi @khmakoto Thanks for your reply.
Figured out that the title was missing on the button as we did not pass the year aria labels in the strings.
After adding the required strings the issue was resolved.
